repairnator/repairnator-scanner/src/main/java/fr/inria/spirals/repairnator/scanner/Launcher.java

```diff
@@ -227,6 +227,10 @@ private void initConfig(JSAPResult arguments) {
         this.config.setLookToDate(lookToDate);
         this.config.setBearsMode(BearsMode.valueOf(arguments.getString("bearsMode").toUpperCase()));
         this.config.setBearsDelimiter(arguments.getBoolean("bearsDelimiter"));
+        this.config.setPipelineMode(arguments.getString("pipelinemode"));
```

Comment:
Better to follow the original naming convention, i.e., pipelineMode. Same for the following arguments.

Reply:
Hi :) , this is a bit weird since at the RTscanner and pipeline, almost all options are lowercased :) .

Reply:
Then it is a inconsistency issue with respect to these files. Because all previous options in this `Launcher` are `lowerCamelCase`. I still suggest to keep the same with the previous code, otherwise it is more weird to have two different naming conventions in one file :)

Reply:
Agree :) .